tests/test_lock.py

```diff
@@ -262,10 +293,37 @@ def test_lock_load(multiprocessing, num_workers, timeout, expected):
 
     assert len(result) == expected
 
-    # Windows removes the lock files
-    if not sys.platform == 'win32':
-        assert os.path.exists(DB_LOCK_PATH)
-        assert os.path.exists(DB_FLAVOR_LOCK_PATH)
+    assert not os.path.exists(DB_LOCK_PATH)
+    assert not os.path.exists(DB_FLAVOR_LOCK_PATH)
+
+
+@pytest.mark.parametrize(
+    'fixture_set_repositories',
+    ['crash-file-system'],
+    indirect=True,
+)
+def test_lock_load_crash(fixture_set_repositories):
+
+    repositories_tmp = audb.config.REPOSITORIES
+
+    audb.config.REPOSITORIES = [
+        audb.Repository(
+            name=pytest.REPOSITORY_NAME,
+            host=pytest.FILE_SYSTEM_HOST,
+            backend='crash-file-system',
+        ),
+    ]
```

Comment:
This should be handled by `fixture_set_repositories` as done in the other tests.

Reply:
Ups. that's a left-over and can be removed of course. Will remove it...

Reply:
Removed it and further simplified the code by using a fixture to test if lock files are removed.